app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ActiveRecord::Base
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def five_last_comments
+    comments.order('created_at').limit(5)
+  end
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
+    user.increment!(:posts_counter)
+  end
```

Comment:
[OPTIONAL]
- Same here. I would recommend making the `update_posts_counter` private method since we are calling this method only from inside the class and with this it is more secure.